submit-job-cli: add exit code documentation (#3552)
ovh/docs: pages/platform/data-processing/37_HOWTO_submit-job-cli/guide.en-asia.md

$ ./ovh-spark-submit --projectid yourProjectId --upload ./spark-examples.jar --class org.apache.spark.examples.SparkPi --driver-cores 1 --driver-memory 4G --executor-cores 1 --executor-memory 4G --num-executors 1 swift://odp/spark-examples.jar 1000
```

> #### Optionally configure your job ExitCode

Reviewer: ```suggestion
#### Optional - Configure your ExitCode job
```

> When your job execution status is complete, by default the ExitCode returns by the CLI Spark-Submit will be zero.

Reviewer: ```suggestion
When your job execution status is complete, by default the ExitCode returned by the CLI Spark-Submit will be zero.
```

> In some case you may want to return an exitcode other than 0, if you do once your job is processed the CLI will return with your code.

Reviewer: ```suggestion
In some cases you may want the returned exitcode  to be different from 0. If you do this once your job is processed, the CLI will return your code.
```

> By example a job counting a number of error will run successfully and will ends with the exit code 0.

Reviewer: ```suggestion
For example, a job counting a number of errors will run successfully and will end with the exit code 0.
```
